templates/echo/files/README.md

```diff
@@ -1,27 +1,18 @@
-# Echo template for Platform.sh
+# Echo for Platform.sh
 
-This project provides a template application using the Echo web framework on Platform.sh. It is primarily an example, although could be used as the starting point for a real project.
+This template builds the Echo framework for Platform.sh.  It includes a minimalist application skeleton for demonstration, but you are free to alter it as needed.
 
-## Starting a new project
+## Services
 
-To start a new project based on this template, follow these 3 simple steps:
+* Go 1.12
+* MariaDB 10.2
 
-1. Clone this repository locally.  You may optionally remove the `origin` remote or remove the `.git` directory and re-init the project if you want a clean history.
-
-2. Create a new project through the Platform.sh user interface and select "Import an existing project" when prompted.
-
-3. Run the provided Git commands to add a Platform.sh remote and push the code to the Platform.sh repository.
-
-That's it!  You now have a working "hello world" level project you can build on.
-
-## Structure
+## Customizations
 
 This project relies on Go module support in Go 1.11 and later.  You should commit your `go.mod` and `go.sum` files to Git, but not the `vendor` directory.
 
-Access to Platform.sh environment information (the port to use, database credentials, etc.) is provided via a [Config Reader](https://github.com/platformsh/config-reader-go) library provided by Platform.sh.
-
-This example shows access to both the port on which to listen for incoming HTTP requests and to a MariaDB database.
-
-## Using as a reference
+The following files and additions make the framework work.  If using this project as a reference for your own existing project, replicate the changes below to your project.
 
-You can also use this repository as a reference for your own projects, and borrow whatever code is needed. The most important parts are the `.platform.app.yaml` file and the `.platform` directory.  You will also need to install the Platform.sh bridge library, `github.com/platformsh/gohelper`.
+* The `.platform.app.yaml`, `.platform/services.yaml`, and `.platform/routes.yaml` files have been added.  These provide Platform.sh-specific configuration and are present in all projects on Platform.sh.  You may customize them as you see fit.
```

Comment:
could we put these file with the link?

Reply:
Possibly. My question is, once this gets shown in the Console, what happens with those links?  Relative links to within the repository work fine on GitHub but when shown in the UI they'd most likely break.  I'd like to confirm with Accounts on what we can do here before doing so.